src/mcif/utils.py

```diff
@@ -34,7 +34,7 @@ def resolve_reference(
         return Path(reference_path)
     version = version if version is not None else __benchmark_version__
     url = _HF_BASE_REPO + f"resolve/{version}/" + \
-        f"MCIF{version}.{track}.{language}.ref.xml.gz?download=true"
+        f"MCIF.{track}.{language}.ref.xml.gz?download=true"
```

Comment:
```suggestion
    if version == '1.0':
        file_name = f"MCIF{version}.{track}.{language}.ref.xml.gz"
    else:
        file_name = f"MCIF.{track}.{language}.ref.xml.gz"
    url = _HF_BASE_REPO + f"resolve/{version}/{file_name}?download=true"
```

Reply:
if you commit this, it should work. Also please add 1.1 to the corresponding test.